otb-tracker/bin

```diff
@@ -13,8 +13,9 @@ shift 2
 export OTB_TRACKER_INTERFACE
 export OTB_TRACKER_CALL
 export OTB_TRACKER_HOST
+export OTB_TRACKER_INITIAL_TIMEOUT
```

Comment:
I do, because I double timeout at each retry. So at the end of the loop, timeout must be reset to its initial value. This is temporary and will be overwritten in next PR (Add_dynamic_timeout)